mirgecom/steppers.py

```diff
@@ -68,6 +71,7 @@ def euler_flow_stepper(parameters, ctx_factory=cl.create_some_context):
     """
     cl_ctx = ctx_factory()
     queue = cl.CommandQueue(cl_ctx)
+    actx = PyOpenCLArrayContext(queue)
```

Comment:
Array context (and queue) creation should be under user control, i.e. in the top-level script. Otherwise you get computations that don't synchronize with anything.

Reply:
Hi. Yes, this function 'euler_flow_stepper' is deprecated, but there is a test or two that need ported over to the "[new way](https://github.com/illinois-ceesd/mirgecom/blob/6b280937340a34a75fb9a5482761394f56a19625/mirgecom/steppers.py#L42)" (tm).

Reply:
OK, let's leave that for #7 then. I'll add a comment there.